packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -242,9 +252,36 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
         return PROOF_MATURITY_DELAY_SECONDS;
     }
 
+    /// @custom:legacy
+    /// @notice Getter for the address of the DisputeGameFactory contract.
+    /// @return Address of the DisputeGameFactory contract.
+    function disputeGameFactory() public view returns (IDisputeGameFactory) {
+        return anchorStateRegistry.disputeGameFactory();
+    }
```

Comment:
Why is this labeled legacy? Since we use it in this contract. Also a nit in that we don't seem usually document self-evident return values
```suggestion
    /// @notice Getter for the address of the DisputeGameFactory contract.
    function disputeGameFactory() public view returns (IDisputeGameFactory) {
        return anchorStateRegistry.disputeGameFactory();
    }
```